development/build/transforms/remove-fenced-code.test.js

```diff
@@ -36,6 +39,10 @@ describe('build/transforms/remove-fenced-code', () => {
       lintTransformedFileMock.mockImplementation(() => Promise.resolve());
     });
 
+    afterEach(() => {
```

Comment:
Why was this addition needed?

Reply:
@danjm Because the previously used jest development config had resetMocks enabled. The main jest config does not have this enabled, so we manually have to reset the mocks. In an ideal world, instead of resetting the mocks manually, we would enable resetMocks in the jest config. However, that would take a considerable amount of time, as we would need to migrate the main test suite to support resetMocks.